### ISSUE TITLE
Require display features option?

### ISSUE THREAD

**Issue by actuallymentor** [daxko/universal-ga#2]
Hello,

Is it possible to do the equivalent of

```js
ga( 'require' , 'displayfeatures')
```

Using this module?

**Comment by scurker** [comment]
~~Yes, you should be able to do this with `.set()`.~~

Nevermind, I just looked at this and realized that it's not currently supported. In the meantime, you should be able to call `ga('require', 'displayfeatures')` as long as `.initialize()` has already been called.

**Comment by actuallymentor** [comment]
This seems to not work according to the GA debugger:

![screen shot 2017-02-23 at 23 41 45](https://cloud.githubusercontent.com/assets/9071382/23282504/be8b19e8-fa21-11e6-8507-41d1a9435cba.png)

EDIT: I responded before reading your edit, sorry :)

**Comment by scurker** [comment]
@actuallymentor Yes, there definitely needs to be a better way to handle plugins. Right now it's a missing feature.

**Comment by scurker** [comment]
This is now available with [v1.1.0](https://github.com/daxko/universal-ga/releases/tag/v1.1.0).

```js
var analytics = require('analytics');
analytics.plugin('displayfeatures');
```

**Comment by actuallymentor** [comment]
Awesome, thank you!
